### PR TITLE
Schinagl.LinkShellextension version 3.9.3.5

### PR DESCRIPTION
**Pull request by schinagl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/89774)

### CODE REVIEW

**Comment by schinagl** [comment]
The odd thing is, this is a Nullsoft installer and it really has /S implemented to run silent
So .... I guess I did something wrong here

File: manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
+PackageLocale: en-US
+Publisher: Schinagl
```
Comment (mdanish-kh) [inline manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml:7]:
```suggestion
Publisher: Hermann Schinagl
```
This must match Control Panel / Apps&Features
image

File: manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
+PackageLocale: en-US
+Publisher: Schinagl
+PackageName: Link Shellextension
+License: MIT License
+Copyright: Copyright © 1999-2020
+ShortDescription: Link Shellextension
```
Comment (mdanish-kh) [inline manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml:11]:
```suggestion
ShortDescription: Link Shell Extension provides for the creation of Hardlinks, Junctions, Volume Mountpoints, and Symbolic Links, a folder cloning process that utilizes Hardlinks / Symbolic Links and a copy process taking care of Junctions, Symbolic Links, and Hardlinks.
```
ShortDescription should be meaningful; this is what I got from the website and I've tried to shorten it to meet the maximum 256 characters requirement. Please commit if you agree with this description.

File: manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
+PackageLocale: en-US
+Publisher: Schinagl
+PackageName: Link Shellextension
```
Comment (mdanish-kh) [inline manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml:8]:
```suggestion
PackageName: Link Shell Extension
```
This must match Control Panel / Apps&Features

File: manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
+PackageLocale: en-US
+Publisher: Schinagl
+PackageName: Link Shellextension
+License: MIT License
+Copyright: Copyright © 1999-2020
```
Comment (mdanish-kh) [inline manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml:10]:
```suggestion
Copyright: Copyright © 1999-2021 Hermann Schinagl
```
I believe this is a typo? This is what I got from your previous PR, please commit if you agree with this change.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
> The odd thing is, this is a Nullsoft installer and it really has /S implemented to run silent So .... I guess I did something wrong here
> 
> Maybe the hint helps: It is a ShellExtension and needs to restart explorer.exe. It kill explorere.exe and then restarts it. Problem?
 
The installer does install silently on my end. Could be the issue you described that's messing with the pipelines as I've noticed many PRs have previously been created for this package and none passed the pipelines. If so, maybe @stephengillie can add a waiver for this package.

**Review by mdanish-kh — APPROVED** [review]

**Comment by schinagl** [comment]
### General
Upfront: I am the developer of LinkshellExtension not a third party who wants to get it favorite tool into a package manager

> During installation testing, this application failed to install without user input. Did you forget to add Silent or SilentWithProgress switches?

I did and it is /S as specified in the .yaml files

> This can also happen when a dependency is missing. You can test with https://github.com/microsoft/winget-pkgs#test-your-manifest.

LSE is dependent on the [vc15 runtimes](https://aka.ms/vs/15/release/vc_redist.x64.exe), since it is compiled with VS2017. These dependencies should be available on a system, but if not how do I specify them?
> 
> Please investigate a fix and resubmit the Pull Request.
> 
> You may also try using the [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) to determine the proper installer type. If the installer type is MSIX, MSI, or a known installer technology like NullSoft, Inno, etc. the wingetcreate tool can detect them and the winget client will know what switches to pass. If it's a .exe installer of an unknown type, you will need to search to determine the proper switches for Silent and SilentWithProgress.
> 

The .yaml files have been created via 

`wingetcreate new`

### Some more hints
- It is a NSIS installer version 2.14
- The NSIS .exe header is compiled by myself
- The NSIS .exe header of the installer is slightly modified, should you go for known checksums. 
- Reason for modification: Nsis generates a 32bit .exe, but the installer needs to make changes to the 64 bit registry since it is a shell extension.
- All NSIS modifications can be found on [LSE gitlab site](https://gitlab.com/schinagl/link-shell-extension)
- It installs silently out of the box on my side and also on the moderators side (see above)
- The exact same installer makes it through the [chocolate checks and pipes](https://community.chocolatey.org/packages/LinkShellExtension)
- The NSIS installer kills explorer during an installation and restarts it afterwards. This is the only way of getting a shellextension into life. Maybe your pipe doesn't like this

I case you can't spot the reason, can you please send me the logs from the pipe where it fails?

Thx Hermann

File: manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
```
Comment (mdanish-kh) [inline manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.installer.yaml:5]:
```suggestion
PackageVersion: 3.9.3.5
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
@schinagl Sincerest apologies as I forgot to check that this package already exists in the repository with the PackageIdentifier `HermannSchinagl.LinkShellExtension` https://github.com/microsoft/winget-pkgs/tree/master/manifests/h/HermannSchinagl/LinkShellExtension/3.9.3.5. I'll move your portable `ln` package to this directory too if you agree with the PackageIdentifier.

**Comment by schinagl** [comment]
> @schinagl Sincerest apologies as I forgot to check that this package already exists in the repository with the PackageIdentifier `HermannSchinagl.LinkShellExtension` https://github.com/microsoft/winget-pkgs/tree/master/manifests/h/HermannSchinagl/LinkShellExtension/3.9.3.5. I'll move your portable `ln` package to this directory too if you agree with the PackageIdentifier.

☺️. Let's do it that way. 
Thx a lot for your excellent help!

It seems someone already added it but didn't tell me. 

Anyhow will update my website to reflect that change. 

Thanks!

**Comment by mdanish-kh** [comment]
My pleasure :) 


> > @schinagl Sincerest apologies as I forgot to check that this package already exists in the repository with the PackageIdentifier `HermannSchinagl.LinkShellExtension` https://github.com/microsoft/winget-pkgs/tree/master/manifests/h/HermannSchinagl/LinkShellExtension/3.9.3.5. I'll move your portable `ln` package to this directory too if you agree with the PackageIdentifier.
> 
> ☺️. Let's do it that way. Thx a lot for your excellent help!
> 
> It seems someone already added it but didn't tell me.
> 
> Anyhow will update my website to reflect that change.
> 
> Thanks!

My pleasure :) I'll close this PR and move the portable `ln` package under the PackageIdentifier `HermannSchinagl.ln`

**Comment by mdanish-kh** [comment]
Close with reason: Package already exists under manifests/h/HermannSchinagl/LinkShellExtension;

**Pull request closed without merging**